gensim/utils.py

```diff
@@ -1168,22 +1175,28 @@ def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
     Python 2.6.2
     Added extra KeyboardInterrupt handling
     """
-    try:
-        logger.debug("COMMAND: %s %s", popenargs, kwargs)
-        process = subprocess.Popen(stdout=stdout, *popenargs, **kwargs)
-        output, unused_err = process.communicate()
-        retcode = process.poll()
-        if retcode:
-            cmd = kwargs.get("args")
-            if cmd is None:
-                cmd = popenargs[0]
-            error = subprocess.CalledProcessError(retcode, cmd)
-            error.output = output
-            raise error
-        return output
-    except KeyboardInterrupt:
-        process.terminate()
-        raise
+    logger.debug("COMMAND: %s %s", popenargs, kwargs)
+    command = kwargs.get('args')[0]
+    usr_bin_alias = '/usr/bin/' + command
+    if not os.path.exists(command) and os.path.exists(usr_bin_alias) and os.path.isfile(usr_bin_alias):
+        command = usr_bin_alias
+    if not os.path.exists(command):
```

Comment:
Need to add tests (system independent) for all raises case.

Reply:
About the windows problem: as far as i remember, we ( me and @tmylk, Moscow, Summer 2017 you were there too :) ) decided that there are so many operating systems that it would not be possible to add functionality for testing all of them.

Reply:
Times change :) Now we can test it with Appveyor (Windows). I see many users who use `gensim.models.wrappers` (i.e. use `check_output` implicitly) on windows, for this reason, I really worried about win-compatibility in this PR.